diff -u old/include/ClientGUIScrolledPanelsManagement.py new/include/ClientGUIScrolledPanelsManagement.py
--- old/include/ClientGUIScrolledPanelsManagement.py	2019-11-17 01:16:40.866092493 +0100
+++ new/include/ClientGUIScrolledPanelsManagement.py	2019-11-17 01:19:43.902194585 +0100
@@ -1709,11 +1709,11 @@
             
             self._verify_regular_https = QW.QCheckBox( general )
             
-            self._network_timeout = QP.MakeQSpinBox( general, min=3, max=300 )
+            self._network_timeout = QP.MakeQSpinBox( general, min=1 )
             self._network_timeout.setToolTip( 'If a network connection cannot be made in this duration or, if once started, it experiences uninterrupted inactivity for six times this duration, it will be abandoned.' )
             
-            self._max_network_jobs = QP.MakeQSpinBox( general, min=1, max=30 )
-            self._max_network_jobs_per_domain = QP.MakeQSpinBox( general, min=1, max=5 )
+            self._max_network_jobs = QP.MakeQSpinBox( general, min=1 )
+            self._max_network_jobs_per_domain = QP.MakeQSpinBox( general, min=1 )
             
             #
             
@@ -1854,9 +1854,9 @@
             self._show_new_on_file_seed_short_summary = QW.QCheckBox( misc )
             self._show_deleted_on_file_seed_short_summary = QW.QCheckBox( misc )
             
-            self._subscription_network_error_delay = ClientGUITime.TimeDeltaButton( misc, min = 600, days = True, hours = True, minutes = True )
-            self._subscription_other_error_delay = ClientGUITime.TimeDeltaButton( misc, min = 600, days = True, hours = True, minutes = True )
-            self._downloader_network_error_delay = ClientGUITime.TimeDeltaButton( misc, min = 600, days = True, hours = True, minutes = True )
+            self._subscription_network_error_delay = ClientGUITime.TimeDeltaButton( misc, min = 0, days = True, hours = True, minutes = True )
+            self._subscription_other_error_delay = ClientGUITime.TimeDeltaButton( misc, min = 0, days = True, hours = True, minutes = True )
+            self._downloader_network_error_delay = ClientGUITime.TimeDeltaButton( misc, min = 0, days = True, hours = True, minutes = True )
             
             #
             
diff -u old/include/ClientGUITime.py new/include/ClientGUITime.py
--- old/include/ClientGUITime.py	2019-11-17 01:16:40.866092493 +0100
+++ new/include/ClientGUITime.py	2019-11-17 01:20:24.041396568 +0100
@@ -55,15 +55,15 @@
         
         self._intended_files_per_check = QP.MakeQSpinBox( self._reactive_check_panel, min=1, max=1000 )
         
-        self._never_faster_than = TimeDeltaCtrl( self._reactive_check_panel, min = 30, days = True, hours = True, minutes = True, seconds = True )
+        self._never_faster_than = TimeDeltaCtrl( self._reactive_check_panel, min = 1, days = True, hours = True, minutes = True, seconds = True )
         
-        self._never_slower_than = TimeDeltaCtrl( self._reactive_check_panel, min = 600, days = True, hours = True, minutes = True )
+        self._never_slower_than = TimeDeltaCtrl( self._reactive_check_panel, min = 1, days = True, hours = True, minutes = True )
         
         #
         
         self._static_check_panel = ClientGUICommon.StaticBox( self, 'static checking' )
         
-        self._flat_check_period = TimeDeltaCtrl( self._static_check_panel, min = 180, days = True, hours = True, minutes = True )
+        self._flat_check_period = TimeDeltaCtrl( self._static_check_panel, min = 1, days = True, hours = True, minutes = True )
         
         #
         
